### DISCUSSION TITLE
求教 AX6 遇到内存泄漏/重启

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#110] [Q&A/答疑解惑] [answered]
抱歉打扰大佬了，因为需要WiFi，被这个问题困扰有点严重，所以来发QA请教

内核是5.10.110 通过开关组件最后得出来是WIFI导致的，关掉WiFi后暂时就没有重启且空闲内存一直较稳定在153M左右

不知道这个问题是否方便解决，如果可以，我能提供哪些log来帮助解决？

另外听说5.15内核在这方面有所改善（？），如果我不使用nss和其它软件，只使用openclash，我可否考虑使用5.15内核的openwrt来观察情况WiFi情况

**Comment by Boos4721** [comment, marked as answer]
1G RAM, OR DO IT.

**Reply by ghost** [reply]
o(╥﹏╥)o 我们这边封了，暂时没法改RAM，请问如果用5.15的内核需要添加什么config

**Reply by Boos4721** [reply]
> o(╥﹏╥)o 我们这边封了，暂时没法改RAM，请问如果用5.15的内核需要添加什么config

*DO IT*

**Reply by ghost** [reply]
> > o(╥﹏╥)o 我们这边封了，暂时没法改RAM，请问如果用5.15的内核需要添加什么config
> 
> _DO IT_

谢谢大佬，最后还是换了闭源，5.15改善并不大

**Reply by flyfishes** [reply]
换了哪个闭源？
